lib/reek/configuration/app_configuration.rb

```diff
@@ -86,7 +84,7 @@ def load(options)
       end
 
       def handle_exclude_paths(paths)
-        self.exclude_paths = paths.map do |path|
+        @exclude_paths = paths.map do |path|
```

Comment:
I don't get it. In https://github.com/troessner/reek/issues/612 we had agreed on "use instance vars in the constructor and for memoization and getters / setters for everything else", hadn't we?